### PR TITLE
Add Python Solution to CSES Playlist

### PR DESCRIPTION
**Pull request by MartinLwx**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/4_Gold/Sliding_Window.mdx [hunk 1]
```diff
@@ -143,6 +143,38 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+def main():
+	"""This code is put in a function for performance reasons."""
```
Comment (SansPapyrus683) [inline content/4_Gold/Sliding_Window.mdx:150]: does it tle outside of main?

File: content/4_Gold/Sliding_Window.mdx [hunk 2]
```diff
@@ -143,6 +143,38 @@ int main() {
...
+	n = int(input().strip())
+	nums = [int(v) for v in input().split(" ")]
+
+	ans = -float("inf")
+	left, right = 0, 0
+	unique_songs = set()
+	while right < n:
+		# Invariant: all the songs in `unique_songs` are unique in each iteration.
+		# To maintain this invariant, shrink the window before inserting nums[right].
```
Comment (SansPapyrus683) [inline content/4_Gold/Sliding_Window.mdx:159]:
```suggestion
		# Notice that all the songs in `unique_songs` are unique in each iteration.
		# We keep this property by shrinking the window before inserting nums[right].
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
yeah this is fine

File: content/4_Gold/Sliding_Window.mdx (hunk 1 above)
Comment (MartinLwx) [reply content/4_Gold/Sliding_Window.mdx]: yeah

File: content/4_Gold/Sliding_Window.mdx (hunk 2 above)
Comment (MartinLwx) [reply content/4_Gold/Sliding_Window.mdx]: Done!

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
FIRE

**Pull request merged**